CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.0.0 (2018/12/04)
+
+Fixes:
+
+* upgrade aws-sdk v3. use aws-sdk-ec2.
```

Comment:
Re-worded to better English at 
https://github.com/sonots/ec2-host/commit/cf9dd3bcf95bccce64d554c3c3d951ed0b40055e